### ISSUE TITLE
Executing ffprobe fails with ValueError: could not convert string to float

### ISSUE THREAD

**Issue by guiweber** [jonghwanhyeon/python-ffmpeg#42]
When executing ffprobe through python-ffmpeg on some MP4 files I encounter the following error:
> ValueError: could not convert string to float: '23.976@GPAC0.5.1-DEV-rev4283'

However, running ffprobe directly on the same file works fine. 

It looks like python-ffmpeg is trying to extract some progress statistics from the ffprobe output. In this case, from the `Title` field of the video stream.

![image](https://github.com/jonghwanhyeon/python-ffmpeg/assets/7825368/317889c4-c1b4-48b7-8370-0557b1279e03)

  ```
File "...\.venv\Lib\site-packages\ffmpeg\ffmpeg.py", line 192, in execute
    raise exception
  File "C:\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...\.venv\Lib\site-packages\ffmpeg\ffmpeg.py", line 253, in _handle_stderr
    self.emit("stderr", line.decode())
  File "...\.venv\Lib\site-packages\pyee\base.py", line 208, in emit
    handled = self._call_handlers(event, args, kwargs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...\.venv\Lib\site-packages\pyee\base.py", line 184, in _call_handlers
    self._emit_run(f, args, kwargs)
  File "...\.venv\Lib\site-packages\pyee\base.py", line 160, in _emit_run
    f(*args, **kwargs)
  File "...\.venv\Lib\site-packages\ffmpeg\progress.py", line 37, in _on_stderr
    statistics = Statistics.from_line(line)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "...\.venv\Lib\site-packages\ffmpeg\statistics.py", line 41, in from_line
    fields = {key: _field_factory[key](value) for key, value in statistics.items()}
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: '23.976@GPAC0.5.1-DEV-rev4283'
```

**Comment by jonghwanhyeon** [comment]
I did not expect such a wide range of corner cases 😂 I just published v2.0.9. Thanks for helping me!

**Comment by guiweber** [comment]
Yeah it can be tricky to handle them all haha. Thanks! :)
